Exercise_7/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace Exercise_7
+{
+    class Program
+    {
+        static void Swap<T>(ref T item1, ref T item2)
+        {
+            T temp;
+            temp = item1;
+            item1 = item2;
+            item2 = temp;
+        }
+        static void Sort<T>(T[] list)
+        {
+            for (var i = 0; i < list.Length - 1; i++)
+            {
+                for (var j = i + 1; j < list.Length; j++)
+                {
+                    if (((ICompare) list[i]).IsBigger((ICompare) list[j]))
```

Comment:
You should use [Constraints](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/generics/constraints-on-type-parameters) or [this](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/where-generic-type-constraint) instead.

Reply:
Done